documents/openapi/openapi.yaml

```diff
@@ -2224,6 +2226,8 @@ components:
           type: string
           format: date-time
           nullable: true
+        segments_count:
+          type: integer
```

Comment:
This isn't being displayed in the swagger editor for the GET /notifications sms example 200 response. Not sure that this is in the correct location in the file.

Reply:
see above for how to add this to the example.

Reply:
I added the new columns to the "sms" and "email" example in both spec files.